pkg/config/config_test.go

```diff
@@ -515,45 +515,31 @@ func (s *MySuite) TestDeploymentName(c *C) {
 
 	// Is deployment_name a valid string?
 	bp.Vars.Set("deployment_name", cty.StringVal("yellow"))
-	dn, err := bp.DeploymentName()
-	c.Assert(dn, Equals, "yellow")
-	c.Assert(err, IsNil)
+	c.Check(checkDeploymentName(bp), IsNil)
```

Comment:
Here and below for all tests.
It seems we test if checkDeploymentName function returns correct error or not. It would be good to add a assert to check if the deployment name is correctly set or not. Like how we had before.

```
c.Assert(dn, Equals, "yellow")
```